### PR TITLE
Class mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/inventory/ClearableInventory.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS afy net/minecraft/inventory/ClearableInventory
```
Comment (asiekierka) [inline mappings/net/minecraft/inventory/ClearableInventory.mapping]: Inventory extends afy, not the other way around.

Comment (Runemoro) [reply mappings/net/minecraft/inventory/ClearableInventory.mapping]: I know, it's not an inventory, but rather an interface implemented by things that have an inventory that can be cleared. It has a single `clear()` method. `ClearableInventoryOwner` maybe?

Comment (asiekierka) [reply mappings/net/minecraft/inventory/ClearableInventory.mapping]: Uh. Does the clear() method *require* or *imply* that the cleared thing is an inventory?

Comment (Runemoro) [reply mappings/net/minecraft/inventory/ClearableInventory.mapping]: The clear method is called:
 - On block entities when using the /clone, /fill, /setblock commands and from a few blocks
 - Directly on inventories from containers and a few other places

Comment (asiekierka) [reply mappings/net/minecraft/inventory/ClearableInventory.mapping]: Yes I'd argue this is more like just _Clearable_ or _ClearableState_ or something

Comment (Runemoro) [reply mappings/net/minecraft/inventory/ClearableInventory.mapping]: Renamed it to `Clearable`.

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/structure/StructurePieceWithDimensions.mapping [hunk 2]
```diff
@@ -0,0 +1,15 @@
+CLASS cdr net/minecraft/structure/StructurePieceWithDimensions
+	FIELD a width I
+	FIELD b height I
+	FIELD c depth I
+	FIELD d hPos I
```
Comment (Gegy) [inline mappings/net/minecraft/structure/StructurePieceWithDimensions.mapping:5]: hPos?

Comment (Runemoro) [reply mappings/net/minecraft/structure/StructurePieceWithDimensions.mapping:5]: It's the NBT name

Comment (asiekierka) [reply mappings/net/minecraft/structure/StructurePieceWithDimensions.mapping:5]: I think it's fine if it's the NBT name, myself

**Review by modmuss50 — APPROVED** [review]
Looks good

**Pull request merged**